cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -455,7 +454,7 @@ export class Application {
             .contains(this.name)
             .closest(trTag)
             .within(() => {
-                clickWithin("#row-actions", button);
+                click('button[aria-label="Kebab toggle"]');
```

Comment:
This locator is already  defined in views/applicationinventory.view.ts. So, this could be replaced.

Reply:
Fixed